frontend/store/fees.ts

```diff
@@ -31,8 +31,12 @@ export const actions = {
         if (!network) {
             return;
         }
-        const fees = await getApproximateTransactionFees(network);
-        commit('setFees', fees);
+        try {
+            const fees = await getApproximateTransactionFees(network);
+            commit('setFees', fees);
+        } catch (error) {
+            console.warn(`Fees calculation error: ${error.message}`);
```

Comment:
nit/q: Why is this just a warning? Aren't the fees quite essential to the system?

Reply:
This is function is called on the interval, populating the store. If it fails ones (and this free endpoints sometimes fail)
- We don't want to show notification to the user since it's not something they initiated or can influence
- If it fails once in a while it doesn't matter: the previous values will stay within the store